doc/assembling.rst

```diff
@@ -0,0 +1,43 @@
+====================================
+Assembling: joining multiple tables
+====================================
+
+.. currentmodule:: skrub
+
+Assembling is the process of collecting and joining together tables. 
+Good analytics requires including as much information as possible,
+often from different sources.
+skrub allows you to join tables on keys of different types
+(string, numerical, datetime) with imprecise correspondance.
+
+Fuzzy joining tables
+---------------------
+
+Joining two dataframes is hard as the corresponding keys are often different.
+
+The :func:`fuzzy_join` uses similarity scores to join tables on one or more
+related columns. Furthermore, it adapts the fuzzy matching based on the
+column type (string, numerical or datetime).
+Using the similarity score, bad matches are easily singled out and can be
+dropped or replaced.
+
+In sum, equivalent to pandas.merge, the :func:`fuzzy_join` 
```

Comment:
We need to use intersphinx, to link to pandas. It's set up in https://github.com/skrub-data/skrub/blob/main/doc/conf.py#L277 . It should work
```suggestion
In sum, equivalent to :func:`pandas.merge`, the :func:`fuzzy_join` 
```